src/network/ConnectionForward.ts

```diff
@@ -239,8 +239,9 @@ class ConnectionForward extends Connection {
 
   protected startTimeout() {
     this.timeout = setTimeout(() => {
+      this.logger.warn('ConnectionForward timing out');
```

Comment:
What does this mesage convey?

Reply:
Removed this for now, as this should be done in #304 (error chaining) and https://github.com/MatrixAI/js-logger/issues/3 (structured logging). The combination of the 2 should make these errors more obvious. I've also embedded the commentary for each error into the description of each exception class now in `network/errors` so that way we get useful feedback.